src/AbpHelper.Core/Steps/Abp/EntityParserStep.cs

```diff
@@ -123,5 +128,42 @@ protected override async Task<ActivityExecutionResult> OnExecuteAsync(WorkflowEx
                 throw;
             }
         }
+
+        private string GetDescription(CSharpSyntaxNode propertyDeclarationSyntax)
+        {
+            return propertyDeclarationSyntax
+                .Descendants<AttributeSyntax>()
+                .Where(p => p.Name.ToString() == "Description" || p.Name.ToString() == "DescriptionAttribute")
+                .Select(p => p.ArgumentList?.Arguments.Select(a => a.ToString().Trim('"')).JoinAsString(";"))
+                .JoinAsString(";");
+        }
+
+        private List<SyntaxKind> documentKinds = new List<SyntaxKind>
+        {
+            //SyntaxKind.DocumentationCommentExteriorTrivia,
+            //SyntaxKind.EndOfDocumentationCommentToken,
+            //SyntaxKind.MultiLineDocumentationCommentTrivia,
+            SyntaxKind.SingleLineDocumentationCommentTrivia
+        };
+
+        private string GetDocument(CSharpSyntaxNode propertyDeclarationSyntax)
+        {
+            return propertyDeclarationSyntax.GetLeadingTrivia()
+                      .Where(p => documentKinds.Contains(p.Kind()))
+                      .Select(p => p.ToString()
+                                    .Replace("/// <summary>\r\n", string.Empty)
+                                    .Replace("/// <summary>", string.Empty)
+                                    .Replace("<summary>\r\n", string.Empty)
+                                    .Replace("<summary>", string.Empty)
+                                    .Replace("/// </summary>\r\n", string.Empty)
+                                    .Replace("/// </summary>", string.Empty)
+                                    .Replace("</summary>\r\n", string.Empty)
+                                    .Replace("</summary>", string.Empty)
+                                    .Trim()
+                                    .Replace("/// ", string.Empty)
+                                    .Replace("///", string.Empty)
+                                    .Replace("\r\n", string.Empty))
+                      .JoinAsString(";");
```

Comment:
Can we just use something like `SingleLineDocumentationCommentTrivia.GetText()` here?